spec/liquid_tags/link_tag_spec.rb

```diff
@@ -19,12 +19,12 @@
     create(:article, user_id: user.id, title: "Hello & Hi <3 <script>", tags: "tag")
   end
 
-  def generate_new_liquid(slug)
+  def generate_new_liquid(slug:)
```

Comment:
I converted it to symbol to make it similar to change done in https://github.com/forem/forem/pull/10437#discussion_r495221635